stores/libsql/src/storage/index.test.ts

```diff
@@ -13,3 +14,75 @@ const mastra = new Mastra({
 });
 
 createTestSuite(mastra.getStorage()!);
+
+// Additional test to check createdAt and updatedAt columns
+describe('LibSQLStore createdAt/updatedAt columns', () => {
```

Comment:
Can you add these tests to `@internal/storage-test-utils';` instead so all storage adatpers can have them